site/docs/v1/tech/themes/_theme-troubleshooting.adoc

```diff
@@ -1,3 +1,23 @@
-If you happen to get into a situation where you have edited a template and it is causing errors that are preventing you from logging in, you can override the use of the UI templates to render a login form that lets you log in. 
 
-To do this, open your browser and access your FusionAuth admin UI. This will redirect you to the broken `/oauth2/authorize` page. Click in your browsers address bar and scroll to the end. Finally, add the String `&bypassTheme=true` to the end of the URL and hit the Enter key. This should render the default login page that ships with FusionAuth and allow you to log in and fix any errors you have.
+// This expects to be brought in under a == heading, otherwise jekyll will complain.
+
+=== Theme Errors Prevent Login
+
+If you happen to get into a situation where you have edited a template and it is causing errors that are preventing you from logging in, you can override the use of the UI templates to render a login form that lets you log in. To do this: 
+
+* Open your browser and access your FusionAuth admin UI. 
+* This will redirect you to the broken `/oauth2/authorize` page. 
+* Click in your browser's address bar and scroll to the end. 
+* Add the String `&bypassTheme=true` to the end of the URL and hit the Enter key. 
+
+This should render the default login page that ships with FusionAuth and allow you to log in and fix any errors you have.
+
+=== Default Theme Used Incorrectly
+
+Anytime a request is made to a themed page and we are unable to identify the tenant, the default tenant will be used. This includes, but is not limited to:
+
+* The root page `/` when a `client_id` or `tenantId` is not provided.
+* Any themed pages such as /password/forgot when a `client_id` is not provided.
+* Edge case error conditions where we have lost the context for the application or tenant.
+
+If you see the default theme unexpectedly, check to make sure you are passing enough information to the page that it can determine the applicable application or tenant.
```

Comment:
maybe... ?
If you see the default theme unexpectedly, ensure the page is passed the appropriate application or tenant information required to render your themed page.